cluster/clean.sh

```diff
@@ -31,11 +31,19 @@ _kubectl delete ds -l "kubevirt.io" -n kube-system --cascade=false --grace-perio
 _kubectl delete pods -n kube-system -l="kubevirt.io=libvirt" --force --grace-period 0 2>/dev/null || :
 _kubectl delete pods -n kube-system -l="kubevirt.io=virt-handler" --force --grace-period 0 2>/dev/null || :
 
-# Delete everything, no matter if release, devel or infra
-_kubectl delete -f ${MANIFESTS_OUT_DIR}/ -R --grace-period 1 2>/dev/null || :
-
-# Delete exposures
-_kubectl delete services -l "kubevirt.io" -n kube-system
+# Delete all traces of kubevirt
+namespaces=(default kube-system)
+for i in ${namespaces[@]}; do
+    kubectl -n ${i} delete deployment -l 'kubevirt.io'
```

Comment:
If we're wanting to ensure this cleans up all VMs and VM pods, we'd want to delete replica sets here as well.

Reply:
Added.